scripts/updateapilist.py

```diff
@@ -95,16 +195,24 @@ class Extractor:
     def client_for_repo(self, repo_slug) -> Optional[CloudClient]:
         path = self.path_format.format(repo_slug=repo_slug)
         url = f"{RAW_CONTENT_BASE_URL}/{path}/{REPO_METADATA_FILENAME}"
-        response = requests.get(url)
-        if response.status_code != requests.codes.ok:
-            return
-
-        return CloudClient(response.json())
+        try:
```

Comment:
You have this pattern in multiple places. Factor it out into a helper function: `get_jason(url, params=None)`. It can set the headers (since they seem to be the same in all calls), log any errors, and decode JSON, returning the results.

Reply:
addressed.